index.html

```diff
@@ -15,10 +15,14 @@
 				<div class="logo"> 
 					<a href="#">George M'sapenda</a>
 				</div>
-					<button class="btn">
-						<img src="/images/Icon - Menu.png" alt="menu bar icon">
-					</button>                    
-				</div>
+				<button class="btn">
+					<img src="/images/Icon - Menu.png" alt="menu bar icon">
+				</button> 
+				<div id="desktop_nav">
+					<a href="#">Portifolio</a>
+					<a href="#">About</a>
+					<a href="#">Contact</a>
+				</div>					    					         			
```

Comment:
- [ ] You have done an outstanding job making the hamburger menu visible only for mobile screens and hiding it for desktop ones and adding the list of navigation links 🎊🙌🏽. However, there are some issues 👇🏽:

1. The image for the hamburger menu for mobile devices is broken and not showing as you can see in the screenshot below. Kindly fix this issue by giving the correct `src="./images/Icon - Menu.png"`.
![image](https://user-images.githubusercontent.com/43172164/200054227-242514f1-1edb-4d9f-9d45-9142231ef8a6.png)
2. Please consider using an **unordered list** `<ul><li></li></ul>` tags for your navigation buttons 🦋, that way you have the correct semantics in your code 📎. **_You will end up with something like this_** 👇🏽
![image](https://user-images.githubusercontent.com/43172164/200054838-d29a67c5-73a7-466f-8df9-1ee2d8185b6a.png)

Reply:
Hi Yazino. Thanks for the timely review. I will use an unordered list. However, from my local repository, the hamburger menu is visible and `src="/images/Icon - Menu.png"` is correct.